specification/resources/resource-manager/Microsoft.Resources/stable/2017-05-10/resources.json

```diff
@@ -2046,6 +2361,10 @@
     },
     "Deployment": {
       "properties": {
+        "location": {
```

Comment:
We cant make this change in existing API-version. Existing SDKs will not be able to deserialize.

Reply:
Right. I'm adding it to a new api-version.